### ISSUE TITLE
Add `<<` operator to specify local config

### ISSUE THREAD

**Issue by ebronasco** [ebronasco/proppy#7]
The new operator would allow you to specify additional parameters for the call. For example, the following code that is used currently in a use-case of `lamoo`:
```
+Return(
        {
            'generate_options': {
                'callback': stop_on('User: '),
            },
        }
    ) \
    | model
```
can be replaced by
```
model << {'callback': stop_on('User: ')}
```

**Comment by ebronasco** [comment]
Add a `Partial` operation that wraps an operation and appends a dict to the inputs of it.

**Comment by ebronasco** [comment]
It is enough to make it a feature of the syntax tree.

**Comment by ebronasco** [comment]
Fixed in #9
